SPECS/cassandra-cpp-driver/cassandra-cpp-driver.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Signatures": {
+    "2.16.0.tar.gz": "35b0f4bac3d17fef47e28611dbeb51e07639395e957f23f4a3cc60770db1ab9c"
```

Comment:
Please use something with a slightly clearer name. I'd recommend `cassandra-cpp-driver-2.16.0.tar.gz`. With just versions for names, we run into a risk of collisions with other tarballs.

To make sure the build takes the right file, suffix `Source0` URL with `#/%{name}-%{version}.tar.gz`. No spaces. This tricks `rpmbuild` to take `%{name}-%{version}.tar.gz` as the expected source file name, while using `rpmspec` to print the source will still return the correct URL.